redbot/cogs/general/general.py

```diff
@@ -234,38 +240,210 @@ def __init__(self):
+            count = 1
+            for emoji, value in online_stats.items():
+                try:
+                    num = len([m for m in guild.members if value(m)])
+                except Exception as error:
+                    print(error)
+                    continue
+                else:
+                    member_msg += f"{emoji} {bold(humanize_number(num))} " + (
+                        "\n" if count % 2 == 0 else ""
+                    )
+                count += 1
+
+            vc_regions = {
+                "vip-us-east": _("__VIP__ US East ") + "\U0001F1FA\U0001F1F8",
+                "vip-us-west": _("__VIP__ US West ") + "\U0001F1FA\U0001F1F8",
+                "vip-amsterdam": _("__VIP__ Amsterdam ") + "\U0001F1F3\U0001F1F1",
+                "eu-west": _("EU West ") + "\U0001F1EA\U0001F1FA",
+                "eu-central": _("EU Central ") + "\U0001F1EA\U0001F1FA",
+                "europe": _("Europe ") + "\U0001F1EA\U0001F1FA",
+                "london": _("London ") + "\U0001F1EC\U0001F1E7",
+                "frankfurt": _("Frankfurt ") + "\U0001F1E9\U0001F1EA",
+                "amsterdam": _("Amsterdam ") + "\U0001F1F3\U0001F1F1",
+                "us-west": _("US West ") + "\U0001F1FA\U0001F1F8",
+                "us-east": _("US East ") + "\U0001F1FA\U0001F1F8",
+                "us-south": _("US South ") + "\U0001F1FA\U0001F1F8",
+                "us-central": _("US Central ") + "\U0001F1FA\U0001F1F8",
+                "singapore": _("Singapore ") + "\U0001F1F8\U0001F1EC",
+                "sydney": _("Sydney ") + "\U0001F1E6\U0001F1FA",
+                "brazil": _("Brazil ") + "\U0001F1E7\U0001F1F7",
+                "hongkong": _("Hong Kong ") + "\U0001F1ED\U0001F1F0",
+                "russia": _("Russia ") + "\U0001F1F7\U0001F1FA",
+                "japan": _("Japan ") + "\U0001F1EF\U0001F1F5",
+                "southafrica": _("South Africa ") + "\U0001F1FF\U0001F1E6",
+                "india": _("India ") + "\U0001F1EE\U0001F1F3",
+                "dubai": _("Dubai ") + "\U0001F1E6\U0001F1EA",
+                "south-korea": _("South Korea ") + "\U0001f1f0\U0001f1f7",
+            }
+            verif = {
+                "none": _("0 - None"),
+                "low": _("1 - Low"),
+                "medium": _("2 - Medium"),
+                "high": _("3 - Hard"),
+                "extreme": _("4 - Extreme"),
+            }
+
+            features = {
+                "PARTNERED": _("Partnered"),
+                "VERIFIED": _("Verified"),
+                "DISCOVERABLE": _("Server Discovery"),
+                "PUBLIC": _("Public"),
+                "INVITE_SPLASH": _("Splash Invite"),
+                "VIP_REGIONS": _("VIP Voice Servers"),
+                "VANITY_URL": _("Vanity URL"),
+                "MORE_EMOJI": _("More Emojis"),
+                "COMMERCE": _("Commerce"),
+                "LURKABLE": _("Lurkable"),
```

Comment:
LURKABLE no longer exists
```suggestion
                "COMMERCE": _("Commerce"),
```